rareapi/views/subscription.py

```diff
@@ -25,7 +28,50 @@ def list(self, request):
             subscriptions, many=True, context={'request': request})
         return Response(serializer.data)
 
+    def create(self, request):
+        """Handle POST operations; 
+        will be used when user is on author profile and hits subscribe;
+        only info needed in request is author ID"""
 
+        follower = RareUser.objects.get(user=request.auth.user)
+        author = RareUser.objects.get(pk=request.data["author"])
```

Comment:
could we change author to author_id in the request data. just to be consistent with other files and more descriptive